build.sh

```diff
@@ -4,4 +4,21 @@ if [ "$#" -ne 1 ]; then
     exit 1
 fi
 
-git archive --format=zip -o doofinder-$1.zip --prefix doofinder/ $1
+cd ..
+
+mkdir _build
+cp -R doofinder-prestashop _build
+mv _build/doofinder-prestashop _build/doofinder
+
+cd _build
+
+rm doofinder/docker-compose.yml
+rm doofinder/composer.json
+rm -rf doofinder/.git
+rm -rf doofinder/.github
+rm doofinder/.git*
+
+zip -r doofinder-$1.zip doofinder
+mv doofinder-$1.zip ..
+cd ..
+rm -rf _build
```

Comment:
Why are you changing this?
The git archive command generates the zip ready to push the release

Reply:
Yes, but doing it that way some unwanted files are being added to the compressed file, like the `composer.json` and `docker-compose.yml`

Reply:
I've added the excluded files in the `.gitattributes` file